test/minitest/test_minitest_test.rb

```diff
@@ -1100,6 +1100,16 @@ def test_autorun_does_not_affect_fork_exit_status
     Process.waitpid(fork { exit 42 })
     assert_equal 42, $?.exitstatus
   end
+
+  def test_autorun_optionally_can_affect_fork_exit_status
+    @assertion_count = 0
+    skip "windows doesn't have skip" unless Process.respond_to?(:fork)
```

Comment:
I assume this is what was meant
```suggestion
    skip "windows doesn't have fork" unless Process.respond_to?(:fork)
```

Reply:
gah! yes. I can do that independent of this branch

Reply:
(does anything else ruby runs on not have fork?)

Reply:
(rebase)

Reply:
> (does anything else ruby runs on not have fork?)

Yes, JRuby and TruffleRuby

Reply:
> (rebase)

Done.